### ISSUE TITLE
support deepseq-1.5 (GHC 9.8)

### ISSUE THREAD

**Issue by frasertweedale** [sw17ch/data-clist#27]
GHC 9.8 bundles deepseq-1.5, which is excluded by data-clist upper bound:

```
 [__6] next goal: data-clist (dependency of brick)
[__6] rejecting: data-clist-0.2, data-clist-0.1.2.3, data-clist-0.1.2.2, data-clist-0.1.2.1, data-clist-0.1.2.0, data-clist-0.1.1.0, data-clist-0.1.0.0 (conflict: pretty => deepseq==1.5.0.0/installed-e7ed, data-clist => deepseq>=1.1 && <1.5)
[__6] rejecting: data-clist-0.0.7.4 (conflict: brick => data-clist>=0.1)
[__6] skipping: data-clist-0.0.7.3, data-clist-0.0.7.2, data-clist-0.0.7.1, data-clist-0.0.7, data-clist-0.0.6, data-clist-0.0.5, data-clist-0.0.4, data-clist-0.0.3, data-clist-0.0.2, data-clist-0.0.1 (has the same characteristics that caused the previous version to fail: excluded by constraint '>=0.1' from 'brick')
```

**Comment by jeremyjh** [comment]
Fixed in 02198c7dfe21b91a52fab0b808b389c2a103733e.

**Comment by frasertweedale** [comment]
Thank you!
